src/IdeHelperServiceProvider.php

```diff
@@ -75,8 +78,11 @@ function ($app) use ($localViewFactory) {
 
         $this->app->singleton(
             'command.ide-helper.models',
-            function ($app) {
-                return new ModelsCommand($app['files']);
+            function (Container $app): ModelsCommand {
```

Comment:
Do we even need to register it? Laravel probably does this already right?

Reply:
Ah, you mean just this?
```php
$this->app->singleton('command.ide-helper.models', ModelsCommand::class);`
```
Should work as well I think!

Reply:
Yeah or maybe not even that.